src/AccessibilityInsights/MainWindow.xaml.cs

```diff
@@ -704,10 +704,32 @@ private static void RestoreConfigurationAsync(/*Action callback = null*/)
                 var serializedConfigsDict = appConfig.IssueReporterSerializedConfigs;
                 Dictionary<Guid, string> configsDictionary = JsonConvert.DeserializeObject<Dictionary<Guid, string>>(serializedConfigsDict);
                 configsDictionary.TryGetValue(selectedIssueReporterGuid, out string serializedConfig);
-                BugReporter.RestoreConfigurationAsync(serializedConfig);
+                await BugReporter.RestoreConfigurationAsync(serializedConfig).ConfigureAwait(false);
+                Dispatcher.Invoke(() =>
+                {
+                    UpdateMainWindowConnectionFields();
+                });
             }
         }
 
+        /// <summary>
+        /// Update sign in logo and tooltip
+        /// </summary>
+        internal void UpdateMainWindowConnectionFields()
+        {
+            bool isConfigured = BugReporter.IssueReporting != null && BugReporter.IsConnected;
+            
+            // Main window UI changes
+            vmAvatar.ByteData = isConfigured ? BugReporter.Logo : null;
```

Comment:
Based on the figma, our logos are going to be fabric icons. We may want to expose a GlyphName string instead of an `IEnumerable<byte>` from the IssueReporter now and remove `vmAvatar`. Thoughts?

Reply:
I'd prefer to see an enum of supported values in the interface defined in the Extensions assembly. That lets us still remain loosely coupled, but with a well-defined mechanism for extending the model in the future. One note: Since it's possible to cast any int to an enum, the code that converts from the enum to the actual glyph would need have a default case to map nonsensical values to some predetermined default

---
In reply to: [267126925](https://github.com/Microsoft/accessibility-insights-windows/pull/194#discussion_r267126925) [](ancestors = 267126925)

Reply:
Can fabric icons be converted to a IEnumerable<byte> form? If yes we should go with that or ask for these icons to be provided to us in a image format. We cannot opt for fabric icons as not all the extensions will have one in the fabric icons set.

Reply:
Fabric icons are currently implemented in the DesktopUI assembly, and the issue filing assembly is allowed to have direct dependencies **only** on the Extensions definition and the Win32 assemblies. That's part of having the loose coupling that is part of the extensions design.

---
In reply to: [267137502](https://github.com/Microsoft/accessibility-insights-windows/pull/194#discussion_r267137502) [](ancestors = 267137502)

Reply:
@flyingUnderTheRadar The only reason we are using `IEnumerable<byte>` is because that is what the ADO profile pic is given to us as. Since we are no longer using that, there is no reason to retain its format. 
> We cannot opt for fabric icons as not all the extensions will have one in the fabric icons set.
One other option would be to take a control and display it here. I would be fine with crossing this bridge when we get to it and using the GlyphName for now.

@DaveTryon I've noticed that we have a few contorls (FabricIcon and MessageDialog specifically) which are a. Useful in the extensions and b. not dependent on anything in AI-Win. Should we consider creating a new project which allows us to share decoupled controls like this accross extensions?

Reply:
I would actually argue that other extensions (3rd party) might not necessarily have representative icons in office fabric so just accepting a generic image would have been great. We discussed this with Risa and she mentioned that we should go ahead with Office Fabric icons for now and that we will change when that case shows up.

Looking into how we can make this change.

Reply:
Ended up following the enum approach.